### PR TITLE
[PUB-2998] Add orgs data to composer

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
- Add orgs data to composer
- Grab orgs data instead of user data in composer queue limit error
- Add plan and role features data to org parser

## Context & Notes
Next: Replace user references (isBusinessUser, features) by orgs data in the composer
https://buffer.atlassian.net/browse/PUB-2998

## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/server/parsers/src/orgParser.js [hunk 1]
```diff
@@ -33,7 +33,25 @@ module.exports = orgData => ({
...
+  canReconnectChannels: orgData.isAdmin,
 
   // Upgrade/ Trial Paths
   showUpgradeToProCta: orgData.planBase === 'free',
+  showUpgradeToBusinessCta: orgData.planBase === 'pro',
+  shouldShowUpgradeButton:
+    orgData.isOwner &&
+    (orgData.planBase === 'free' ||
+      orgData.planBase === 'pro' ||
+      orgData.plan === 'solo_premium_business' ||
```
Comment (albennett) [inline packages/server/parsers/src/orgParser.js:55]: We don't still offer the `solo_premium_business` plan, right?

File: packages/composer/composer/action-creators/AppActionCreators.js [hunk 2]
```diff
@@ -240,10 +240,13 @@ const AppActionCreators = {
...
           if (unsuccessfulResponse.code === UpgradeErrorCodes.queueLimit) {
-            const userData = AppStore.getUserData();
-            const { isFreeUser, isBusinessUser, canStartProTrial } = userData;
+            const organizations = AppStore.getOrganizationsData();
+            const {
+              showUpgradeToBusinessCta,
+              showUpgradeToProCta,
+            } = organizations.selected;
             const scope = `${NotificationScopes.PROFILE_QUEUE_LIMIT}-${unsuccessfulResponse.serviceName}`;
-            const source = 'queue_limit';
```
Comment (albennett) [inline packages/composer/composer/action-creators/AppActionCreators.js:246]: Was this just an unused variable?

**Review by albennett — APPROVED** [review]
Great refactor, esp with the composer 💯

File: packages/composer/composer/action-creators/AppActionCreators.js (hunk 2 above)
Comment (anafilipadealmeida) [reply packages/composer/composer/action-creators/AppActionCreators.js:246]: Yap 😅

File: packages/server/parsers/src/orgParser.js (hunk 1 above)
Comment (anafilipadealmeida) [reply packages/server/parsers/src/orgParser.js:55]: Don't we @albennett? I was keeping the same logic already in place. Also, the plans package is full of references about the solo_premium_plan 🤔

Comment (albennett) [reply packages/server/parsers/src/orgParser.js:55]: I was thinking it was from a past experiment Jenny did 🤔 But I may be wrong! I think regardless we'd need to support the plan as people will have signed up with it, even if we no longer offer it new customers.

**Pull request merged**